spec/spec_helper.rb

```diff
@@ -48,7 +48,7 @@ module BlockScore
     Kernel.srand config.seed
 
     config.around do |example|
-      Timeout.timeout(10, &example)
+      Timeout.timeout(6, &example)
```

Comment:
Why adjust this timeout?

OT I should mention that when I was working from a coffeeshop I had to locally edit this in order to allow the specs to pass due to a less than favourable speed connection.